gallery/visualtests/nxToast.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+const { setupBrowser } = require('./testUtils');
+
+describe('NxToast', function() {
+  const openButton = '.nx-btn-bar .nx-btn--secondary';
+  const toastContainer = '.nx-toast-container';
+
+  describe('with global sidebar and header', function() {
+    const {
+      checkFullPageScreenshot,
+      getPage,
+      wait,
+      waitAndGetElements,
+      a11yTest
+    } = setupBrowser('#/NxToastComplexLayoutExample', false);
+
+    async function launchToasts(toastsContainer, buttonSelector) {
+
+      const [openToastBtn] = await waitAndGetElements(buttonSelector);
+      await openToastBtn.click();
+      await openToastBtn.click();
+
+      // wait for animation
+      await wait(500);
+
+      const [container] = await waitAndGetElements(toastsContainer);
+      return container;
+    }
+
+    describe('NxToastContainer', function() {
```

Comment:
Are these nested `describe('NxToastContainer'` block valuable?

Reply:
> Are these nested `describe('NxToastContainer'` block valuable?

At the time I thought it should describe the container being positioned, but unnecessary. Removed here : [8e399c9](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/8e399c93d842d791e4e9290f050592ba121b9f92)